notebooks/oracle2bq/OracleToBigQuery_notebook.ipynb

```diff
@@ -0,0 +1,888 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "1db5371a-8f16-47b7-bcc7-5af386e9b6d8",
+   "metadata": {},
+   "source": [
+    "# <center>Oracle to BigQuery"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "98acd907",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Copyright 2022 Google LLC\n",
+    "#\n",
+    "# Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "# you may not use this file except in compliance with the License.\n",
+    "# You may obtain a copy of the License at\n",
+    "#\n",
+    "#     https://www.apache.org/licenses/LICENSE-2.0\n",
+    "#\n",
+    "# Unless required by applicable law or agreed to in writing, software\n",
+    "# distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "# See the License for the specific language governing permissions and\n",
+    "# limitations under the License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dd944742",
+   "metadata": {},
+   "source": [
+    "#### References\n",
+    "\n",
+    "- [DataprocPySparkBatchOp reference](https://google-cloud-pipeline-components.readthedocs.io/en/google-cloud-pipeline-components-1.0.0/google_cloud_pipeline_components.experimental.dataproc.html)\n",
+    "- [Kubeflow SDK Overview](https://www.kubeflow.org/docs/components/pipelines/sdk/sdk-overview/)\n",
+    "- [Dataproc Serverless in Vertex AI Pipelines tutorial](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/ml_ops/stage3/get_started_with_dataproc_serverless_pipeline_components.ipynb)\n",
+    "- [Build a Vertex AI Pipeline](https://cloud.google.com/vertex-ai/docs/pipelines/build-pipeline)\n",
+    "\n",
+    "This notebook is built to run a Vertex AI User-Managed Notebook using the default Compute Engine Service Account.  \n",
+    "Check the Dataproc Serverless in Vertex AI Pipelines tutorial linked above to learn how to setup a different Service Account.  \n",
+    "\n",
+    "#### Permissions\n",
+    "\n",
+    "Make sure that the service account used to run the notebook has the following roles:\n",
+    "\n",
+    "- roles/aiplatform.serviceAgent\n",
```

Comment:
missing BQ permissions in the list.

Reply:
can you please test and ensure that this list of roles is correct, by defining a service account which has only these roles.